emission/tests/analysisTests/intakeTests/TestUserStat.py

```diff
@@ -44,7 +44,7 @@ def setUp(self):
             edb.get_profile_db().insert_one({"user_id": self.testUUID})
 
         #etc.runIntakePipeline(self.testUUID)
-        etc.runIntakePipeline(self.testUUID)
+        epi.run_intake_pipeline_for_user(self.testUUID, skip_if_no_new_data = False)
```

Comment:
This test is failing because `epi.run_intake_pipeline_for_user` never calls `eaurs.get_and_store_pipeline_independent_user_stats`, and these tests 

I think with `etc.runIntakePipeline`, everything will run including the pipeline and pipeline-independent stats

```suggestion
        etc.runIntakePipeline(self.testUUID)
```

Reply:
Yup, just made that change and pushed it. I spent some time trying to figure out why we had commented out `etc.runIntakePipeline` in favor of `epi.run_intake_pipeline_for_user` in the first place, but it was part of the initial commit